****************************************************************
*                      SLURM Batch System                      *
*           IN2P3 Computing Centre, Villeurbanne FR            *
****************************************************************
* Date:                 Wed Mar 20 19:31:07 CET 2024
* Job name:             scaling_rel
* Job id:               4152023
* User:                 cpayerne
* Account:              lsst
* Submit host:          jns-cpayerne.cc.in2p3.fr
* Partition:            htc
* Quality of service:   normal
* Nodelist:             ccwslurm0110
****************************************************************
ln: failed to access ‘/pbs/home/c/cpayerne/.conda/envs/myenv/share/gdb/auto-load//pbs/home/c/cpayerne/.conda/envs/myenv/lib/libarrow.so.1100.0.0-gdb.py’: Input/output error
--- WLxN ---
--- N ---
--- WL ---
--- MxN ---
--- M ---
Traceback (most recent call last):
  File "/pbs/throng/lsst/users/cpayerne/CLCosmo_Sim/cluster_scaling_relation_DC2/modules/run_mcmc_cluster_scaling_relation_from_lensing_profiles.py", line 245, in <module>
    print(lnL(initial))
  File "/pbs/throng/lsst/users/cpayerne/CLCosmo_Sim/cluster_scaling_relation_DC2/modules/run_mcmc_cluster_scaling_relation_from_lensing_profiles.py", line 178, in lnL
    proxy_mu0, proxy_mulog10m, proxy_sigma0, proxy_sigmalog10m = theta
ValueError: too many values to unpack (expected 4)
****************************************************************
*                      SLURM Batch System                      *
*           IN2P3 Computing Centre, Villeurbanne FR            *
****************************************************************
Date: Wed Mar 20 19:31:18 CET 2024
Job informations can be found using these commands:
Accounting:
sacct -j 4152023
Efficiency:
seff 4152023
****************************************************************
